src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-redis.mdx

```diff
@@ -0,0 +1,174 @@
+
+## Step 1: Sign up for your free account if you haven't already [#signup]
+
+<InlineSignup />
+
+## Step 2: Prerequisites [#prerequisites]
+
+The collector will use the [redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver) component (minimum version recommended v0.83.0), and it requires access to Redis through an endpoint.
+
+## Step 3: Install OpenTelemetry collector [#install-opentelemetry-collector]
+
+Download and install the OpenTelemetry collector following [OpenTelemetry docs](https://opentelemetry.io/docs/collector/getting-started/).
+
+You will need to install [OpenTelemetry Collector Contrib Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib) or other distribution including, at least, the following components:
+
+* [Redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver)
+* [OTLP exporter](https://pkg.go.dev/go.opentelemetry.io/collector/exporter/otlpexporter)
+* [Attribute processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md)
+* [Resource processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourceprocessor/README.md)
+* [Resource detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md)
+
+## Step 4: Configure OpenTelemetry collector [#configure-opentelemetry-collector]
+
+Update or create a new file called `config.yaml` from the example below.
+
+Notice that, since no identificator is attached by the OTel receiver to redis metrics, an attribute processor is needed to decorate metrics with `server.address` and `server.port` metadata. 
+The attributes `server.address` and `server.port` are needed to be able to query and facet data differentiating between Redis servers and to create NewRelic entities. 
+
+Moreover, separate pipelines with different instances of `resource/redis_metrics` are needed in case multiple Redis servers are monitored from a single otel collector to decorate each instance with the corresponding metadata.
+
+
+Replace the `NEW_RELIC_LICENSE_KEY` in the file with your own values. For more info, see the [New Relic License key](/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+
+```yaml
+receivers:
+  redis:
+    endpoint: "{redis_hostname}:{redis_port}"
+    metrics:
+      # Enable redis.maxmemory optional metric
+      redis.maxmemory:
+        enabled: true
+
+processors:
+  # This is needed since redis receiver is not currently adding any means to understand to which redis server the metrics are referring to 
+  attributes/redis_metrics:
+    include:
+      match_type: regexp
+      metric_names:
+        # If you are installing with Helm the regex should be escaped further "redis\\..*"
+        - "redis\..*" 
```

Comment:
```suggestion
        - "redis\\..*"
```

We need to escape it, otherwise, the collector will fail on start-up.

Reply:
The otel config is actually:
```
          match_type: regexp
          metric_names:
          - "redis\..*"
```
on the other hand to pass that config from HELM we need:
```
          match_type: regexp
          metric_names:
            - "redis\\..*"
```

Reply:
We sorted it out off-line, thanks for the input!